lms/static/sass/course/layout/_courseware_header.scss

```diff
@@ -42,19 +42,14 @@
         text-align: center;
         text-decoration: none;
 
-        &:hover,
-        &:focus {
-          color: $uxpl-blue-hover-active;
-          border-bottom: 4px solid $uxpl-blue-hover-active;
-        }
-
+        &,
         &.active {
-          border-bottom: 4px solid $uxpl-blue-base;
-          color: $uxpl-blue-base;
+          border-bottom: 4px solid $uxpl-blue-hover-active;
+          color: $uxpl-blue-hover-active;
 
           &:hover,
           &:focus {
-            color: $uxpl-blue-base;
+            color: $uxpl-blue-hover-active;
```

Comment:
I'm a bit confused here. Isn't this the same color that is defined in the parent rule? Does this mean that there isn't a color change on hover/focus?